integration-tests/README.md

```diff
@@ -122,7 +122,7 @@ In order to be able to run the [performance tests scenarios](src/test/resources/
 - [Hyperfoil v0.21](https://hyperfoil.io/) - Benchmark framework for microservices (*)
 - [webhook-perf-test](https://github.com/afalhambra/webhook-perf-test) - Dummy app to consume cloud events from webhook Processor
 
-There is a dedicated Maven profile named `performance` located in `integration-tests/pom.xml`. There you can see the following system properties needed 
+There is a dedicated Maven profile named `performance` located in `integration-tests/manager-integration-tests/pom.xml`. There you can see the following system properties needed 
```

Comment:
IMHO this readme can be removed, at least for now, as all the info is related to manager integration tests and already included in readme in manager integration tests.